setup.cfg

```diff
@@ -59,9 +60,8 @@ envlist =
     mypy
     # tests against released versions
     py{38,39}-dj{42}-redislatest
-    py{310,311,312}-dj{42,50,51}-redislatest
+    py{310,311,312}-dj{42,50,51,52}-redislatest
     # tests against unreleased versions
-    py311-dj51-redismaster
```

Comment:
shouldn't this be kept with python3.12, django 5.2 and redis master?

I think tox tries to launch the combination but it doesn't use redis master for some reason